### DISCUSSION TITLE
Probleme bouton

### DISCUSSION THREAD

**Discussion by ogiusson** [shanteacontrols/OpenDeck#293] [General]
Bonjour, je n'arrive pas a faire qu'un bouton se désactive quand je relache. C'est toujours une impulsion pour mettre en route, puis réappuyer pour eteindre, je voudrais que quand je relache l'action se termine.
j'ai essayé tout les types de messages, note, cc... et toujours le même probleme.
Quelqu'un peut m'aider svp?
merci

**Comment by paradajz** [comment]
What exactly makes you think French is spoken here?

**Comment by ogiusson** [comment]
sorry, everything was in French ...
so I have a problem with a configuration about a button action, I try to
have the action just when my button is pushed. and not one push on / one
push off.
do you know how can I do that?
thanks.

**Comment by paradajz** [comment]
You can select latching button type if that's what you're referring to.

**Reply by ogiusson** [reply]
Thanks, 
i tried it but not working...

**Reply by ogiusson** [reply]
i see the activity is good, it's latching but not on my DAW
